### DISCUSSION TITLE
Mooney M20R exhaust gas temp

### DISCUSSION THREAD

**Discussion by dieharder** [nguyenquyhy/Flight-Tracker-StreamDeck#196] [Q&A]
Hi,

Did anyone already figure out which unit to use for EGT on the Mooney M20R (Carenado).
I'm using GENERAL_ENG_EXHAUST_GAS_TEMPERATURE__1, which works as expected in other planes, and gives C by default

The dial in the cockpit says Fahrenheit, but:
- Dial ingame says 1367
- Unit fahrenheit gives 1012 in streamdeck
- Unit rankine 1472
- Unit celsius 545
- Unit kelvin 818

![Schermopname (86)](https://user-images.githubusercontent.com/29387056/110022722-93678480-7d2c-11eb-8808-c983a0c98122.png)

Thanks

**Comment by SerpaFox** [comment]
Hi @dieharder,

Did you solve the issue? I have the same problem...

**Reply by dieharder** [reply]
Sadly no, didn't find a solution yet, I just removed that indicator from the Streamdeck and forgot about it.

**Reply by SerpaFox** [reply]
Ok, I opened a support ticket at Carenado. Lets see if they can help us. I hope you don't mind I used part of your message.

Cheers!
